lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py

```diff
@@ -878,53 +888,61 @@ def exec_command(self, command):
             stderr=subprocess.DEVNULL,
         )
 
+    def exec_command_output(self, command: str):
+        """Run a command remotely on magma_dev VM.
+
+        Args:
+            command: command (str) to be executed on remote host
+            e.g. 'sed -i \'s/config1/config2/g\' /etc/magma/mme.yml'
+
+        Returns:
+            output of command execution
+        """
+        data = self._data
+        data["command"] = '"' + command + '"'
+        param_list = shlex.split(self._command.format(**data))
+        return subprocess.check_output(
+            param_list,
+            shell=False,
+        ).decode("utf-8")
+
```

Comment:
It is probably better to move this method further down where it was originally so that the diff is easier to read.

Reply:
This was already merged in #13854 as part of a cleanup PR.